content/management/configuration-api/v3.3/index.mdx

```diff
@@ -888,18 +888,23 @@ curl -X POST \
 
 ### List Bots
 
+Lists all the Bots within a license that were created by the requester. The method behavior differs depending on who created the Bot and who the requester is.
+
+- If a Bot was created **directly by an Agent** (PAT authorization), all Agents within the license will get that Bot in the response, even with the `all` parameter set to `false`.
```

Comment:
all agents will get that bot if they use PAT; if they use token from any app, they won't see the bot